lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -466,7 +467,9 @@ sub _engine_workit_step_2 ($job, $job_settings, $vars, $shared_cache, $callback)
             local $ENV{PERL5OPT} = '';
             # Allow to override isotovideo executable with an arbitrary
             # command line based on a config option
-            exec $job_settings->{ISOTOVIDEO} ? $job_settings->{ISOTOVIDEO} : ('perl', $isotovideo, '-d');
+            my $args = ('-d');
+            push @{$args}, '--color=yes' if $isotovideo_interface_version > 26;
+            exec $job_settings->{ISOTOVIDEO} ? $job_settings->{ISOTOVIDEO} : ('perl', $isotovideo, $args);
```

Comment:
```suggestion
            exec $job_settings->{ISOTOVIDEO} ? $job_settings->{ISOTOVIDEO} : ('perl', $isotovideo, @$args);
```

Reply:
I guess I can just declare it as `@args` 🤔 Interestingly the tests are fine as-is